vxwhatsapp/config.py

```diff
@@ -8,7 +8,7 @@
 WHATSAPP_NUMBER = os.environ.get("WHATSAPP_NUMBER", "none")
 PUBLISH_TIMEOUT = int(os.environ.get("PUBLISH_TIMEOUT", "10"))
 CONCURRENCY = int(os.environ.get("CONCURRENCY", "50"))
-CONSUME_TIMEOUT = float(os.environ.get("CONSUME_TIMEOUT", "10"))
+CONSUME_TIMEOUT = float(os.environ.get("CONSUME_TIMEOUT", "15"))
```

Comment:
If this is just affecting BWise, then we can just change this environment variable there, I don't think we need to change the default unless we're seeing this issue on multiple projects.